environment

```diff
@@ -0,0 +1,44 @@
+# DSS Auth Environment Variables
+#
+# Resolve the location of this file and set DSS_HOME to the root
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
+export DSS_AUTH_HOME="$(cd -P "$(dirname "$SOURCE")" && pwd)"
+
+AWS_DEFAULT_OUTPUT=json
+AWS_DEFAULT_REGION=us-east-1
+GCP_DEFAULT_REGION=us-central1
+GCP_PROJECT_ID="platform-hca"
+DSS_DEPLOYMENT_STAGE="dev"
+DSS_PLATFORM="ucsc-cgp"
+DSS_SECRETS_STORE="${DSS_PLATFORM}/dss"
+
+
+#not sure if these are needed
+OIDC_AUDIENCE=https://dss.dev.ucsc-cgp-redwood.org
+AUTH_URL=https://auth.ucsc.ucsc-cgp-redwood.org
+OPENID_PROVIDER=https://dev-4lyab62k.auth0.com/
+OIDC_EMAIL_CLAIM="${OIDC_AUDIENCE}/email"
+OIDC_GROUP_CLAIM="${OIDC_AUDIENCE}/group"
+
+
+##
+AUTH_TENANT_SECRET_NAME="auth_tenant"
+
+# These are placeholders, please please these in an environment.local to avoid leaks.
```

Comment:
Can split this into an `environment.local.example` in this PR or a future PR

Reply:
i oped to drop that entire file , and use SSM

Reply:
nice, the environment-free revolution inches forward